ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -244,15 +244,10 @@ function getDiseaseNames(keyword) {
       return response.json();
     })
     .then((matches) => {
-      const labeledMatches = fp.filter((elt) => elt.hasOwnProperty('label'))(
-        matches
+      const labeledMatches = matches.filter((elt) =>
```

Comment:
`fp` can be replaced by non-fp `filter()` and `map()` which the linter understands better